aws/resource_aws_volume_attachment_test.go

```diff
@@ -329,10 +329,40 @@ data "aws_availability_zones" "available" {
   }
 }
 
+data "aws_ami" "amzn-ami-minimal-hvm-ebs" {
```

Comment:
Can use ` testAccLatestAmazonLinuxHvmEbsAmiConfig` here

https://github.com/terraform-providers/terraform-provider-aws/blob/dcb00966d9ec8383883ad62750cb2455fedd96f9/aws/resource_aws_instance_test.go#L4379-L4399

which we may want to move into `provider_test.go` as it's applicable to many more resources than just EC2 instances.

Reply:
Good point, I'll replace this for my other relevant PRs as well.